pkg/sdk/poc/main.go

```diff
@@ -5,6 +5,7 @@ package main
 import (
 	"bytes"
 	"fmt"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
```

Comment:
strange that our linter is not complaining here about the import not at the bottom 🔍

Reply:
oh, you are right